packages/vuepress-theme-nimatron/enhanceApp.js

```diff
@@ -0,0 +1,9 @@
+import './styles/index.scss';
+
+export default ({
+  Vue,
+  options,
+  router,
+  siteData
+}) => {
+}
```

Comment:
This file doesn't appear to be used. Can we add this in later once there is a need for it?

Reply:
This is required by VuePress